addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1361,7 +1361,6 @@ export class OdooEditor extends EventTarget {
         return false;
     }
     historySetSelection(step) {
-        this.deselectTable();
```

Comment:
@visp-odoo why we removed this?

Reply:
@shsa-odoo If we not remove this then after hovering one color if you try to hover on another color then it will deselect the table and toolbar gets closed.so, you will see that hover on another color gives you a bug again.

Reply:
I'm not sure this is ok though. The DOM changes when we revert the history, then we either handle the selection or let the browser do it. But the browser doesn't care about our table selections so we might end up with an incorrect table selection. That should be alright in most cases since it should trigger a `selectionchange` event which would end up calling `deselectTable` but that's only if the new selection is contained in the editable area.

So the issue would happen if (1) a history revert broke the selection, (2) we had no way to restore it and (3) it was so broken it ended up outside the editable or was even destroyed.

The question is whether this arrives in reality but I would guess it does because that would explain why I had put that line there :-P

cc @dmo-odoo 

PS: for reference, this is where the line was added: https://github.com/odoo/odoo/pull/92200/commits/85015cca574853437787cc70fc77ed439bcc4d3c#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR1157